owasp-top10-2017-apps/a7/comment-killer/killer/src/components/Blog.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+function blog() {
+    return (
+        <div>
+            <div
+                style={{
+                    backgroundColor: "#D5E3E3",
+                    textAlign: "center",
+                    paddingLeft: 200,
+                    paddingRight: 200,
+                }}
+            >
+                <h3>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Eu lobortis elementum nibh tellus molestie. Vel
+                    pharetra vel turpis nunc eget. Leo vel fringilla est
+                    ullamcorper eget nulla facilisi etiam. Eget lorem dolor sed
+                    viverra ipsum nunc aliquet bibendum. Malesuada fames ac
+                    turpis egestas maecenas pharetra convallis. Dignissim enim
+                    sit amet venenatis. Vitae turpis massa sed elementum.
+                    Suspendisse potenti nullam ac tortor. Id aliquet lobortis
+                    elementum nibh tellus molestie. Vel pharetra vel turpis
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Eu lobortis elementum nibh tellus molestie. Vel
+                    pharetra vel turpis nunc eget. Leo vel fringilla est
+                    ullamcorper eget nulla facilisi etiam. Eget lorem dolor sed
+                    viverra ipsum nunc aliquet bibendum. Malesuada fames ac
+                    turpis egestas maecenas pharetra convallis. Dignissim enim
+                    sit amet venenatis. Vitae turpis massa sed elementum.
+                </h3>
+            </div>
+            <div style={{ textAlign: "right", marginRight: 300 }}>
+                <img
+                    style={{ borderRadius: 1000, height: 100 }}
+                    src="https://i.kym-cdn.com/entries/icons/facebook/000/032/279/Screen_Shot_2019-12-30_at_11.26.24_AM.jpg"
```

Comment:
Can you download this image and add it into this repository? We can reference it later on by doing something similar to:

```html
<img style={{ borderRadius: 1000, height: 100 }} src="images/logo.jpg"
```